comictaggerlib/seriesselectionwindow.py

```diff
@@ -465,24 +465,24 @@ def categorize(result: ComicSeries) -> int:
             self.twList.setItem(row, 0, item)
 
             item_text = str(record.start_year)
-            item = QtWidgets.QTableWidgetItem(item_text)
+            item = QtWidgets.QTableWidgetItem(record.start_year)
             item.setData(QtCore.Qt.ItemDataRole.ToolTipRole, item_text)
+            item.setData(QtCore.Qt.ItemDataRole.DisplayRole, record.start_year)
             item.setFlags(QtCore.Qt.ItemFlag.ItemIsSelectable | QtCore.Qt.ItemFlag.ItemIsEnabled)
             self.twList.setItem(row, 1, item)
 
             item_text = str(record.count_of_issues)
-            item = QtWidgets.QTableWidgetItem(item_text)
+            item = QtWidgets.QTableWidgetItem(record.count_of_issues)
```

Comment:
Same as above: This sets the item [type] and should not be used instead pad to four characters: `f"{record.start_year:04}"`